alf/environments/alf_wrappers.py

```diff
@@ -831,6 +831,116 @@ def time_step_spec(self):
         return self._time_step_spec
 
 
+@alf.configurable
+class CarlaMergedActionWrapper(AlfEnvironmentBaseWrapper):
```

Comment:
This is only useful for carla. It's better to put it in suite_carla

Reply:
Move this wrapper and also CarlaActionWrapper into carla_utils.

Reply:
To make code review easier, I split the updates into to commits: 
1. Code changes: 76749c13ee385835788c7be52b6e4c26f9544dbf
2. Code movements: 83ee1c7fde3f57673d4d5462a8cd2e1ad1b3a886